concepts/modularization/README.md

```diff
@@ -0,0 +1,75 @@
+# Modularization
+
+Kyma provides Kubernetes building blocks. It should be easy to pick only those that are needed for the job and it should be easy to add new blocks to extend Kyma features. With the growing number of components, it is not possible to always install them all anymore. 
+
+- [Independent releases of modules](#independent-releases-of-modules)
+- [Dependencies between components](#dependencies-between-components)
+- [Release channels](#release-channels)
+- [Versioning](#versioning)
+- [Manifest operator](#manifest-operator)
+
+# Independent releases of modules
+With the growing number of components, it is hard to deliver features and fixes quickly and efficiently. Changes in manifests require a new release of Kyma. Operators (reconcilers) are tightly coupled and must be released together. In most cases, new component releases don't involve any API changes and could be delivered in a few minutes. 
+
+
+# Dependencies between components
+Components can depend only on core Kubernetes API or API extensions introduced by other components. Component operators are responsible for checking if the required APIs are available and react properly to the missing dependencies by reducing functionality or even reporting errors. Component owners are responsible for integration tests with their dependencies (with all versions supported in official release channels).
+
+# Release channels
+Release channels let customers balance between available features and stability. The number of channels and their characteristics can be established later. Usually, projects introduce between 2 and 4 channels. Some examples:
```

Comment:
Would different release channels introduce different features?
Do you mean here that optional modules would be enabled on different channels? (i.e Keda )
Or rather that some beta feaures may be available earlier in some release channels ( for early adopters )

Reply:
Yes - new features/modules could be tested in the rapid channel. I will update the concept.